src/components/common/Input.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+import styles from './Input.module.css';
+
+const Input = ({ onChange, ...otherProps }) => {
+    const [value, setValue] = useState('');
+
+    const onChangeValue = (e) => {
+        const value = e.target.value;
+        setValue(value);
+        onChange?.(value);
```

Comment:
what is it?

Reply:
what have are you passing to the Input?

Reply:
Done. 
I added propTypes to make it clear what I'm passing to the Input